tests/licensedcode/data/plugin_license/external_licenses/scan.expected.json

```diff
@@ -0,0 +1,81 @@
+{
+  "headers": [
+    {
+      "tool_name": "scancode-toolkit",
+      "options": {
+        "input": "<path>",
+        "--external-license-directory": "<path>",
+        "--json": "<file>",
+        "--license": true,
+        "--strip-root": true
+      },
+      "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
+      "output_format_version": "2.0.0",
+      "message": null,
+      "errors": [],
+      "warnings": [],
+      "extra_data": {
+        "system_environment": {
```

Comment:
IMHO we should include the list of paths to additional licenses directories (either plain dirs or plugin-contributed) in the output as extra data, otherwise there is no way to know something non-standard was used.